### DISCUSSION TITLE
Why does a related collection get multiple aliases for each of its fields?

### DISCUSSION THREAD

**Discussion by arikw** [directus/directus#17331] [Archive]
I've noticed that in the tabular layout, if I'm adding multiple fields of a related collection, the request's payload has multiple aliases for the same related collection.

For example, when viewing a collection page in the app, a request to `/items/my_table` is dispatched as follows:

![image](https://user-images.githubusercontent.com/7900646/215283492-f9594cd6-61d2-4c11-a32e-7b70d41302bb.png)

**Comment by rijkvanzanten** [comment]
That's to prevent problems from occurring if you request different nested structures for the same top level field, for example `translations` & `translations.language_code` or `translations.author` & `translations.author.name`
